enforcer/src/keystate/keystate_ds_submit_task.c

```diff
@@ -48,7 +48,7 @@ keystate_ds_submit_task_perform(task_type *task)
 		KEY_DATA_DS_AT_PARENT_SUBMIT, KEY_DATA_DS_AT_PARENT_SUBMITTED,
 		(engine_type*)task->context);
 	task_cleanup(task);
-	flush_enforce_task((engine_type*)task->context, 0);
+	flush_enforce_task((engine_type*)task->context, time_now());
```

Comment:
Order has and was always as is afaik.  "has no effect" is /always/ "probably".  I believe the effect is null, and no tests are affected.  Which also says things about the test.  I will revert it, it did no mean the same thing I though it did, as clearly indicated in my commit.